pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> Query and modify dconf settings with schema validation.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
+
+- Display all (non-relocatable) schemas, keys, and values:
+
+`gsettings list-recursively`
+
+- Display all keys and values (default if not set) from one schema:
+
+`gsettings list-recursively {{org.example.schema}}`
+
+- Set the value of a key. Fails if key doesn't exist or value is outside range:
+
+`gsettings set {{org.example.schema}} {{example-key}} {{value}}`
```

Comment:
Asking fellow maintainers: Should we use `example_key` here like we do everywhere or should we use the formatting style of a gsettings key (e.g. `always-show-log-out`)?

Reply:
For what it's worth, [the docs](https://developer.gnome.org/gio/stable/GSettings.html) do specifically say:

>Key names are restricted to lowercase characters, numbers and '-'.

Reply:
That's why I'm not sure. Independent of what the command uses, we generally use snake_case for arguments.